UIAutoMonkey.js

```diff
@@ -23,8 +23,54 @@
 function UIAutoMonkey() {
 		
 	this.config = {
+		//run either by minutesToRun or numberOfEvents. Only one of these can set. (To use minutes you can use config.numberOfEvents = 0)
+		//minutesToRun = 60 * 8; //sample to run for 8 hours.
+		//checkTimeEvery = 60; //how often to check (in events) if minutesToRun has is used. 
 		numberOfEvents: 1000,
 		delayBetweenEvents: 0.05,    // In seconds
+		
+	   /**
+		* sometimes the monkey can fall into UI Holes from which it it is hard to escape. The monkey may then spend an inordinate
+		* amount of time in such holes, neglecting other parts of the application. 
+		*
+		* For example, if a parent Window P has a large image
+		* and clicking on the image opens a child window C from which one exits by tapping a small X on the top right, then until that small X is
+		* tapped we will reamin in C. conditionHandlers offer the developer the option to periodically recognize that we are in C and press the X.
+		*
+		* See buttonHandler.js for a specialized conditionHandler useful when a top level button can be used to escape from a UI hole.
+		*
+	    * conditionHandlers are objects that respond to the following function calls:
```

Comment:
The indentation of these following lines doesn't match the lines above. Can you check to see if there are some tabs/spaces mixed up together?